src/api/auth-identities.tsx

```diff
@@ -74,3 +74,20 @@ export const deleteOIDCIdentities = (
       .catch(reject);
   });
 };
+
+export const fetchIdentityDetails = (
```

Comment:
I think `createFineGrainedTlsIdentity` might be a more appropriate name for this function, wdyt?